octobot_trading/api/exchange.py

```diff
@@ -43,12 +44,13 @@ def get_exchange_manager_from_exchange_name_and_id(exchange_name: str, exchange_
 
 async def get_exchange_available_time_frames(
```

Comment:
Should we name it `get_ccxt_exchange_available_time_frames` as it is only able to fetch ccxt exchange time frames ?

Reply:
right